src/components/PokeCard.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { mockPokemonData } from "../mock/pokeData";
+
+function RenderPokeCard() {
+  return (
+    <div>
+      <h1>{mockPokemonData.name} </h1>
```

Comment:
I recommend you to use destructuring to keep code cleaner and easy to read
```suggestion
    const { name, sprites, video } = mockPockemonData;
      <h1>{name} </h1>
```